tests/containers/run_container_in_k3s.pm

```diff
@@ -0,0 +1,45 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Installs local k3s locally and executes a test
+# to be sure this is working properly
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils;
+use publiccloud::utils;
+use containers::k8s qw(install_k3s apply_manifest wait_for_k8s_job_complete find_pods validate_pod_log);
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    my $provider = get_var("KUBECTL_CLUSTER", "k3s");
```

Comment:
Do we really need different providers in a test run that is explicitly named `run_container_in_k3s`?

Reply:
Yes, I agree, initial code was more general.

Reply:
done